app/src/main/java/fr/free/nrw/commons/review/ReviewImageFragment.java

```diff
@@ -81,17 +81,17 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container,
             case SPAM:
                 question = getString(R.string.review_spam);
                 explanation = getString(R.string.review_spam_explanation);
-                yesButtonText = getString(R.string.review_spam_yes_button_text);
-                noButtonText = getString(R.string.review_spam_no_button_text);
+                yesButtonText = getString(R.string.no);
```

Comment:
can you assign the **yes button** with text for **Yes** instead of **No** and same for **no button**?